config.default

```diff
@@ -326,6 +326,15 @@ movingAveragePoints = 12
 # regionIndicesToPlot should be an integer
 regionIndicesToPlot = 5
 
+# Data source to read for comparison.  There are three options
+# 1 - ERS SSTv3b -- Common in CVDP analysis -- 1854 - 2016
+# 2 - ERS SSTv4 -- Updated version of previous -- 1854 - 2016
+# 3 - HADIsst -- Hadley center analysis -- 1870 - 2016
+# Valid Choices -- Uncomment your choice 
+#observationData = /global/homes/l/lvroekel/HADIsst_nino34.nc
+observationData = /global/homes/l/lvroekel/ERS_SSTv3b_nino34.nc
+#observationData = /global/homes/l/lvroekel/ERS_SSTv4_nino34.nc
```

Comment:
I feel that these should be handled differently.  The path to the subdirectory with the observations should be placed in `[oceanObservations]`.  The file names themselves should either be hardcoded or included as only the basename (without the full path) under `[oceanObservations]`.  Here, you could have a flag with which the user could choose which (if any) of the data sets to plot.  It might make sense to have this variable support a single string or a list of strings, in case you want to plot them all:
```
plotObservations = ['HADIsst', 'ERS SSTv3b', 'ERS SSTv4']
```
would be used to plot all 3.

It might also be useful to have the option of plotting observation time series, observation spectra, or both:
```
plotObsTimeSeries = False
plotObsSpectra = True
```
This would be the default I would suggest, but it is up to you (and @milenaveneziani).